.taprc

```diff
@@ -1,9 +1,4 @@
 jobs: 1
-
-branches: 96
-functions: 100
-lines: 100
-statements: 98
-
+disable-coverage: true
```

Comment:
Isn't it better to impose a fixed limit, so that we're aware when the code coverage drop?

Maybe we can open an issue to increase code coverage if possible.

Reply:
> Maybe we can open an issue to increase code coverage if possible.

Yes

> Isn't it better to impose a fixed limit, so that we're aware when the code coverage drop?

I'm not aware if that exists in tap@20, happy to add it back if it exists

Reply:
> I'm not aware if that exists in tap@20, happy to add it back if so

Yep, they broke a few things.
Maybe it's better to fix coverage in an independent PR, will take a look tomorrow